view/frontend/web/css/styles.css

```diff
@@ -33,6 +36,39 @@
     }
 }
 
+#ui-datepicker-div.bk-datepicker {
```

Comment:
I think it is  better to use the class instead of id for CSS selector .ui-datepicker.bk-datepicker

Reply:
I want a higher specificity so the default styles don't override our changes